project/bintray.sbt

```diff
@@ -1,2 +0,0 @@
-addSbtPlugin("com.eed3si9n" % "bintray-sbt" % "0.3.0-a1934a5457f882053b08cbdab5fd4eb3c2d1285d")
-resolvers += Resolver.url("bintray-eed3si9n-sbt-plugins", url("https://dl.bintray.com/eed3si9n/sbt-plugins/"))(Resolver.ivyStylePatterns)
```

Comment:
dropped, switching to released 0.3.0. I'm pretty sure the version above was the sha of @eed3si9n's changes while it was being pull request reviewed.